lms/djangoapps/teams/static/teams/js/views/edit_team.js

```diff
@@ -124,10 +133,9 @@
                     var validationResult = this.validateTeamData(data);
                     if (validationResult.status === false) {
                         this.showMessage(validationResult.message, validationResult.srMessage);
-                        return;
+                        return $().promise();
```

Comment:
I've always used:

    $.Deferred().resolve().promise()

I don't know which is better idiomatically, but yours is shorter! 

http://stackoverflow.com/questions/25032549/use-of-dummy-and-empty-jquery-promises